lib/services/schema-object-factory.ts

```diff
@@ -204,11 +204,13 @@ export class SchemaObjectFactory {
     if (!propertiesWithType) {
       return '';
     }
+    const extensionProperties = Reflect.getMetadata(DECORATORS.API_EXTENSION, type) || [];
```

Comment:
```suggestion
    const extensionProperties = Reflect.getMetadata(DECORATORS.API_EXTENSION, type) || {};
```
?

Reply:
Thanks. I think the empty array and object spread the same way, but better to use the right type for documentation purposes.